src/libraries/System.Security.Cryptography.Encoding/src/System.Security.Cryptography.Encoding.csproj

```diff
@@ -20,66 +20,42 @@
     <Compile Include="System\Security\Cryptography\OidGroup.cs" />
     <Compile Include="System\Security\Cryptography\PemFields.cs" />
     <Compile Include="System\Security\Cryptography\PemEncoding.cs" />
-    <Compile Include="$(CommonPath)Internal\Cryptography\Helpers.cs"
-             Link="Internal\Cryptography\Helpers.cs" />
-    <Compile Include="$(CommonPath)System\Security\Cryptography\CryptoPool.cs"
-             Link="System\Security\Cryptography\CryptoPool.cs" />
-    <Compile Include="$(CommonPath)System\HexConverter.cs"
-             Link="Common\System\HexConverter.cs" />
+    <Compile Include="$(CommonPath)Internal\Cryptography\Helpers.cs" Link="Internal\Cryptography\Helpers.cs" />
```

Comment:
Please revert all of the changes in this csproj, including the file moves.  It's not necessary to the namespace rename.

Reply:
Done.